observatory-dags/observatory/dags/workflows/crossref_events_telescope.py

```diff
@@ -27,12 +27,12 @@
 import requests
 from airflow.exceptions import AirflowSkipException
 from airflow.models.taskinstance import TaskInstance
-from observatory.platform.telescopes.stream_telescope import (
+from observatory.platform.workflows.stream_workflow import (
```

Comment:
I would prefer to call it stream_telescope, same reason as the snapshot_telescope